src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+/**
+ *  CS2113T Semester 2 AY19/20
+ *  Individual Project
+ *
+ *  Project Duke is a educational software project designed to take you through
+ *      the steps of building a small software incrementally,
+ *      while applying as many Java and SE techniques as possible along the way.
+ *
+ *  The project aims to build a product named Duke, a Personal Assistant Chatbot that
+ *      helps a person to keep track of various things. The name Duke was chosen as a placeholder name,
+ *      in honor of Duke, the Java Mascot. You may give it any other name and personality you wish.
+ *
+ *  @file/s: Duke.java Printer.java Storage.java Task.java
+ *  @author: Tan Zheng Fu Justin
+ */
+
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    static Storage myTasks = new Storage();
+
+    public static void start() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Printer.printStart(logo);
+    }
+
+    public static void exits() {
+        Printer.printLines();
+        Printer.printIndentation();
+        Printer.printBye();
+        Printer.printLines();
+    }
+
+    public static Task readFromUser() {
+        Scanner in = new Scanner(System.in);
+        String userInput;
+
+        userInput = in.nextLine();
+        Task task = new Task(userInput);
+
+        return task;
+    }
+
+    public static String[] parseCommand(String description) {
+        String[] commands = description.split(" ");
+        return commands;
+    }
+
+    public static void echoUntilBye() {
+        boolean isBye = false;
+        while (!isBye) {
+            Task task = readFromUser();
```

Comment:
I think we can create a new Task only after we get the command which requires to do so.Since if the command is "list" or "bye", then there is no need to so.